app/models/cms/email_template.rb

```diff
@@ -50,7 +50,25 @@ def self.reset_templates_path!
   EMAIL_FORMAT = /(?:(#{EMAIL_ADDRESS_FORMAT})|#{EMAIL_WITH_NAME_FORMAT})\s*/i
   EMAILS_FORMAT = /\A#{EMAIL_FORMAT}(,\s*#{EMAIL_FORMAT})*\Z/i
 
-  serialize :options, Hash
+  class HashOrParameters < ::ActiveRecord::Coders::YAMLColumn
+    def load(string)
+      obj = super(string)
+      obj.respond_to?(:to_unsafe_h) ? obj.to_unsafe_h : obj.to_h if obj
+    end
+
+    def dump(object)
+      return object if object.is_a?(String) && assert_valid_value(load(object))
+      obj = object.respond_to?(:to_unsafe_h) ? object.to_unsafe_h : object.to_h if object
```

Comment:
Now it is serialized either as `ActionController::Parameters` or `Hash`

Reply:
yeah.. but we should find where is the saving point that is storing the serializer as ActionController::Parameters